README.md

```diff
@@ -24,15 +24,28 @@ In the challenge for this Sprint, you will demonstrate proficiency by creating a
 
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
-- [ ] What are the required parts of class components?
+- [ ] What are the required parts of class components?  
+    class component requires you to extend from React.Component and create a render function which returns a React element. and the required parts of the class component are constructor ,parent constructor and state.
 
-- [ ] Name at least three lifecycle methods?
+- [ ] Name at least three lifecycle methods? 
+   ComponentDidMount
+   ComponentDidUpdate
+   ComponentUnMount
```

Comment:
Did you mean componentWillUnmount? Also, notice the capitalization. Not a huge deal, but could trip you up later.